packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,70 @@
+import Router from 'express/lib/router'
+import { authMiddleware, validatePost } from '../middleware'
+import { db } from '../store'
+
+const app = Router()
+
+app.get('/', authMiddleware({ admin: true }), async (req, res, next) => {
+  const [regions, cursor] = await db.region.find({}, { limit: 100 })
+  if (req.query.grouped === 'true') {
+    return res.json(regions)
+  }
+
+  const flatOrchList = []
+  regions.forEach((region) => {
+    region.orchestrators.forEach((orch) => {
+      orch.region = region.region
+      flatOrchList.push(orch)
+    })
+  })
+
+  return res.json(flatOrchList)
+})
+
+app.get('/:region', authMiddleware({ admin: true }), async (req, res, next) => {
+  const region = await db.region.get(req.params.region)
+  if (!region) {
+    return res.status(404)
```

Comment:
This will actually hang forever because there's no `res.end()`. How 'bout
```suggestion
    res.status(404)
    return res.json({errors: ["not found"]})
```